usr.bin/grep/util.c

```diff
@@ -223,6 +225,9 @@ procmatch_match(struct mprintc *mc, struct parsec *pc)
 			else
 				break;
 		}
+		pc->matchidx = 0;
```

Comment:
Nit: let's add a blank line above this, please

Reply:
Added.